src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -169,73 +177,38 @@ public static Person createDeletePerson(Person personToEdit, DeletePersonDescrip
     }
 
     /**
-     * Stores the boolean value of the fields that is to be deleted from a person in
-     * the address book.
+     * Stores the boolean value of the fields that is to be deleted from a patient
+     * in
+     * HealthSync.
      */
     public static class DeletePersonDescriptor {
-        private boolean phone;
-        private boolean medicalHistory;
-        private boolean appointment;
-        private boolean email;
-        private boolean address;
-        private boolean tags;
+        private boolean shouldDeleteAppointment;
+        private boolean shoudlDeleteMedicalHistory;
 
         public DeletePersonDescriptor() {
         }
 
-        public void setPhone() {
-            this.phone = true;
-        }
-
-        public boolean getPhone() {
-            return this.phone;
-        }
-
-        public void setEmail() {
-            this.email = true;
-        }
-
-        public boolean getEmail() {
-            return this.email;
-        }
-
-        public void setAddress() {
-            this.address = true;
-        }
-
-        public boolean getAddress() {
-            return this.address;
-        }
-
-        public void setMedicalHistory() {
-            this.medicalHistory = true;
-        }
-
-        public boolean getMedicalHistory() {
-            return this.medicalHistory;
-        }
-
-        public void setAppointment() {
-            this.appointment = true;
+        public void setShouldDeleteMedicalHistory() {
+            this.shoudlDeleteMedicalHistory = true;
         }
 
-        public boolean getAppointment() {
-            return this.appointment;
+        public boolean getShouldDeleteMedicalHistory() {
+            return this.shoudlDeleteMedicalHistory;
```

Comment:
Would it be better to name this as `shouldDeleteMedicalHistory()` or `hasMedicalHistory()`? These getters are boolean methods, so naming it like this may cause coding standard violations.